src/NUnitFramework/framework/Assert.Conditions.cs

```diff
@@ -339,8 +339,8 @@ public static void IsNaN(double? aDouble, string? message, params object?[]? arg
         }
 
         /// <summary>
-        /// Verifies that the double that is passed in is an <code>NaN</code> value. Returns without throwing an
-        /// exception when inside a multiple assert block.
+        /// <para>Verifies that the double that is passed in is an <c>NaN</c> value. Returns without throwing an
+        /// exception when inside a multiple assert block.</para>
```

Comment:
💭 Interesting. The `<para>` warning in this case was actually a red herring. The use of `<code>` made it _think_ there were multiple paragraphs when in reality it should have used `<c>` for just one paragraph.

Reply:
Thanks for pointing that out, that's helpful.

Reply:
Nothing to do with this issue, but that's actually an odd comment. The lack of an exception in a multiple assert block is a feature of __all__ asserts, not just this one.

Reply:
@CharliePoole I added that comment to all assertion methods because of how often the comes up in support questions that these methods can be depended on to throw AssertionException for failures.

Reply:
Ah! I see you did that quite a while back, in fact.